### PR TITLE
fix: improve typing of getAccount, and deprecated incorrectly named parameter

### PR DESCRIPTION
**Pull request by daniel-makerx**

### CODE REVIEW

File: docs/code/modules/index.md [hunk 1]
```diff
@@ -484,7 +484,7 @@ If that code runs against LocalNet then a wallet called `ACCOUNT` will automatic
 
 | Name | Type | Description |
 | :------ | :------ | :------ |
-| `account` | `string` \| { `fundWith?`: [`AlgoAmount`](../classes/types_amount.AlgoAmount.md) ; `name`: `string`  } | The details of the account to get, wither the name identifier (string) or an object with: * `name`: The name identifier of the account * `fundWith`: The amount to fund the account with it it gets created (when targeting LocalNet), if not specified then 1000 Algos will be funded from the dispenser account |
+| `account` | `string` \| { `fundWith?`: [`AlgoAmount`](../classes/types_amount.AlgoAmount.md) ; `name`: `string`  } | The details of the account to get, either the name identifier (string) or an object with: * `name`: The name identifier of the account * `fundWith`: The amount to fund the account with it it gets created (when targeting LocalNet), if not specified then 1000 Algos will be funded from the dispenser account |
```
Comment (aorumbayev) [inline docs/code/modules/index.md:487]: `it it gets` -> `it gets` ?

**Review by aorumbayev — APPROVED** [review]
@daniel-makerx lgmt, approved, just one minor typo noted

File: docs/code/modules/index.md (hunk 1 above)
Comment (daniel-makerx) [inline docs/code/modules/index.md:487]:
```suggestion
| `account` | `string` \| { `fundWith?`: [`AlgoAmount`](../classes/types_amount.AlgoAmount.md) ; `name`: `string`  } | The details of the account to get, either the name identifier (string) or an object with: * `name`: The name identifier of the account * `fundWith`: The amount to fund the account with when it gets created (when targeting LocalNet), if not specified then 1000 Algos will be funded from the dispenser account |
```

Comment (daniel-makerx) [reply docs/code/modules/index.md]: Good spot, found the same typo in a few spots

**Pull request merged**